### ISSUE TITLE
Medium 2.1 has "fatal error" crash with Revive 1.5

### ISSUE THREAD

**Issue by EE128** [LibreVR/Revive#1082]
I've tried every combination of reinstalling and launching and even tried SteamVR beta and the openvr beta. Every time I launch it, it loads for a second then goes to a white screen with a dialog box that says fatal error and sometimes  "Failed to initialize rift"

My specs are:
Windows 7
i7-2600k 3.4Ghz
16gb RAM
Radeon RX570

**Comment by Armada651** [comment]
Are you *sure* you've updated?

**Comment by EE128** [comment]
I'm a little confused on what you're asking. The versions I have are the latest, all were downloaded this morning with fresh installs. Steam, Oculus, revive, Medium, etc

**Comment by neftavazquez** [comment]
> I've tried every combination of reinstalling and launching and even tried SteamVR beta and the openvr beta. Every time I launch it, it loads for a second then goes to a white screen with a dialog box that says fatal error and sometimes "Failed to initialize rift"
> 
> My specs are:
> Windows 7
> i7-2600k 3.4Ghz
> 16gb RAM
> Radeon RX570

I have the same problem, say FATAL ERROR

**Comment by ivanmalvin** [comment]
I have the same problem. Medium crashes instantly without loading. The Medium window is pure white and says Fatal Error / Medium has crashed.

**However** I was able to get Medium to launch by switching to Steam VR **NON-beta.** I was able to reproduce this problem by switching back and forth between Steam VR beta and non-beta. It works with non-beta. Crashes when I have steamvr beta running.

Now that Steam VR beta has motion smoothing I'd really like to be able to stick with the Steam VR beta.

I had been using Steam VR beta with Revive / Medium 2.0 in the past. I haven't used it for a month or two. Maybe this problem is caused by the Medium 2.1 update 2 weeks ago, or the recent Steam VR beta update which added motion smoothing?

Everything is updated on my end which I verified while testing this. While troubleshooting I made sure to try reinstalling Revive (just downloaded the latest new version today Nov 1st), Steam VR is updated, Medium is updated (v2.1)
